src/llmcompressor/pipelines/sequential/helpers.py

```diff
@@ -403,6 +405,46 @@ def match_modules(model: Module, target_names: List[str]) -> Set[Module]:
     )
 
 
+def get_targets_from_modifiers(
```

Comment:
nit: could we update the name to reflect targets are only fetched from sequential modifiers?

Reply:
This function operates on all modifiers. For example, if no targets are given, then the targets are inferred. In the future, this will be a config external to the modifiers entirely